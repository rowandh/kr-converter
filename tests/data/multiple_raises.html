
<iframe width="0" height="0" name="hfrm"></iframe>
<script language="javascript">
	function goProc(mode,uid,infotype,money){
		form = document.form1;
		form.target="hfrm";
		if (money < 1) {
			alert("1보다 큰금액만 입력하실수 있습니다");
			return;
		}
		form.infotype.value=infotype;
		form.money.value=money;
		form.user_id.value=uid;
		form.action="userinmoney_proc.asp";
		
			if(confirm(uid + "에게 " + money + "원 하자처리 하시겠습니까?")){
				form.submit();
			}		

	}
</script>

<style>
		body {background:#f5f5f5;}
		* {padding:0; margin:0;}
		input:focus,
		select:focus {
			outline:none;
		}
		.page-layout {
			height:100%;
			
		}
		
		.page-layout .bar-search {
			display:flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
			gap:0.875rem;
			
			padding:0 0.875rem 0.875rem;
			background:#2e2f39;
		    border-bottom: 1px solid #e6e6f2;
		    box-shadow: 0 1px 4px 0 rgba(82,63,105,.15);
		}
		
		.page-layout .bar-search .title {
			padding-top:0.875rem;
			font-family:'Malgun Gothic';
			color:#c5ff88;
			font-size:1.425rem;
			font-weight:bolder;
			letter-spacing:-2px;
		}
		
		.page-layout .bar-search .title > span {
			font-size:1rem;
			letter-spacing:0;
		}
		
		.page-layout .bar-search > div:nth-child(2) {
			flex-grow:1;
			
			display:inline-flex;
    		flex-wrap: wrap;
    		justify-content: flex-end;
			gap:0.875rem;
			
			padding-top:0.875rem;
			border-top:1px solid #60931e;
		}
		
		.page-layout .bar-search > div:nth-child(2) span {
		    line-height:2rem;
		    color: #fff;
		    font-weight:bold;
		}
		
		.page-layout .bar-search > div:nth-child(2) select,
		.page-layout .bar-search > div:nth-child(2) input {
		    height: calc(2em + 0.8rem + 2px);
		    padding:0 1rem;
		    font-size: 1rem;
		    font-weight: 400;
		    height:2rem;
		    line-height:2rem;
		    color: #ccc;
		    background-color: #141529;
		    background-clip: padding-box;
		    border: 1px solid #141529;
		    border-radius: 0.25rem;
		    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
		}
		
		.page-layout .bar-search > div:nth-child(2) input.btn-serarch {
			color:#fff;
			font-weight:bold;
			background:#73a730 ;
			border:1px solid #629322;
		}
		
		
		.table-area {
			border:1px solid #e6e6f2;
			border-radius:1rem;
			overflow-x:auto;
			background:#fff;
			box-shadow:3px 3px 5px #e7e7e7;
			margin:1rem;
		}
		
		.table-area th {
			padding: 0 0.5rem;
			font-size:1rem;
			height:46px;
			font-weight: 600;
    		color: #2e2f39;
			background:#fff;
			border-bottom:2px solid #e6e6f2;
		}
		
		.table-area th a {
			color:#cf0b0b;
			text-decoration:none;
		}
		
		.table-area td {
			font-size:14px;
			padding:0.425rem 0.75rem;
			color:#71748d;
			white-space:nowrap;
			background:#fff;
			border-bottom:1px solid #e6e6f2;
		}
		/*
		.table-area tr:nth-child(2n) td {
			background:#f4f4f5;
			border-top:1px solid #e6e6f2;
			border-bottom:1px solid #e6e6f2;
		}
		*/
		.table-area th,
		.table-area td {
			border-right:1px solid #e6e6f2;
		}
		
		.table-area table td a {
			display:inline-block;
			
			padding:0 1rem 0 1.425rem;
			line-height:1rem;
			color:#5e8d21;
			font-size:14px;
			font-weight:bold;
			text-decoration:none;
			border-radius:0.312rem;
			background:url('../images/icon-move.png') left 50% no-repeat;
		}
		
		.table-area table td a:hover {
			filter:hue-rotate(275deg);
		}
		
		.table-area table th.double-table,
		.table-area table td.double-table {
			padding:0;
		}
		
		
		.table-area table th.double-table th {
			padding:0 0.75rem;
			border-bottom:none;
		}
		
		.table-area table td.double-table td {
			line-height:1.5rem;
			border:none;
			background:none;
		}
		
		.table-area table td.double-table tr td+td {
			border-left:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr+tr td {
			border-top:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr:nth-child(2n) td {
			background:#f9f9f9;
		}
		
		.btn-history-back {
			padding:0 1rem;
			font-size:1rem;
			height:2.625rem;
			line-height:2.625rem;
			color:#fff;
			font-weight:bold; 
			background:#37a730; 
			border-radius:0.25rem; 
			border:1px solid #629322; 
			box-shadow:1px 1px 0 #629322;
			cursor:pointer;
		}
		
		.btn-history-back:hover {
			filter:brightness(1.1);
			border:1px solid #37a730; 
			box-shadow:1px 1px 0 #37a730;
		}
		
	</style>
	
	<div class="page-layout">
		<div class="bar-search">
			<div class="title">게임기록보기</div>
			<div>
                <div>
					<input type="button" value="닫기" onclick="self.close()" class="btn-serarch" />
                </div>	
			</div>
		</div>
		
		<div class="table-area">
			<table cellspacing="0" width="100%"  border="0" >
				<tr>
                          <th width="" height="34"><p align="center">라운드ID</p></th>
                          <th width="" height="34"><p align="center">시각</p></th>
                          <th width="" height="34"><p align="center">게임 종류</p></th>
                          <th width="" height="34"><p align="center">승자(족보)</p></th>
					<th width="" height="34"><p align="center">이긴금액</p></th>
					<th class="double-table">
						상세정보
					</th>
				</tr>




                            <tr bgcolor="white">
								<td width="56" height="26"><p align="center">&nbsp;15-2-86576952</p></td>
                                <td height="26" ><p align="center">&nbsp;2024-12-27 오후 7:20:56</p></td>
                                <td ><p align="center">&nbsp;홀덤</p></td>
                                <td ><p align="center">&nbsp;<b>3t2gh2j4 (9 트리플)</b></p></td>
<!--								<td ><p align="center">&nbsp;237025</p></td>   -->
								<td ><p align="center">&nbsp;237,025</p></td>
								<td class="double-table">
									
									<table cellspacing="0" width="100%"  border="0">										
										<colgroup>
											<col width="200" />
											<col width="*" />
											<col width="160" />
											<col width="200" />
										</colgroup>
										<tr>
											<th height="34" style="word-break:break-all"><p align="center">참가자</p></th>
											<th height="34" style="word-break:break-all"><p align="center">족보</p></th>
											<th height="34" style="word-break:break-all"><p align="center">변동 금액</p></th>
											<th height="34" style="word-break:break-all"><p align="center">남은 잔액</p></th>
										</tr>
										
										


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>woeirvkd</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣6 ♣2 [6 탑]  (다이) 
<br>* 시작 : [StageNo:86576952] [Credit:200,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:200]
<br>* NICKNAME:[woeirvkd]
<br>* 앤티: -1,000원(199,000원)
<br>* 홀 카드딜: ♣6(44) ♣2(40) [6 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:6 탑(♣6 ♣2)]
<br>* 베팅: 다이 [0](199,000원) - 베팅순서: [0][4]
[214ms]
<br>* 커뮤니티 카드 딜: H(♣6♣2) C (♣9♠A♣10) (<font color='red'>♥</font>8) (<font color='red'>♥</font>J)
<br>* 종료: WinMoney[0원] Credit[199,000원]
<br>* 결과: 패배 [족보:6 탑] [카드:♣6 ♣2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">199,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>ftxijbnridn</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>8 ♠5 [8 탑]  (다이) 
<br>* 시작 : [StageNo:86576952] [Credit:311,675원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[scinnine]
<br>* 앤티: -1,000원(310,675원)
<br>* 홀 카드딜: <font color='red'>◆</font>8(20) ♠5(4) [8 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:8 탑(<font color='red'>◆</font>8 ♠5)]
<br>* 베팅: 다이 [0](310,675원) - 베팅순서: [0][2]
[293ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>8♠5) C (♣9♠A♣10) (<font color='red'>♥</font>8) (<font color='red'>♥</font>J)
<br>* 종료: WinMoney[0원] Credit[310,675원]
<br>* 결과: 패배 [족보:8 탑] [카드:<font color='red'>◆</font>8 ♠5] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">310,675</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>sdhewrj35</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣A ♠J ♣9 ♠A ♣10 <font color='red'>♥</font>8 <font color='red'>♥</font>J [A J 투 페어]  
<br>* 시작 : [StageNo:86576952] [Credit:455,667원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[sdhewrj35]
<br>* 앤티: -1,000원(454,667원)
<br>* 홀 카드딜: ♣A(39) ♠J(10) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(♣A ♠J)]
<br>* 베팅: [레이즈] (10,000원) Credit(444,667원) - 베팅순서: [0][5]
[4250ms]
<br>* 베팅: [레이즈] (444,000원) Credit(667원) - 베팅순서: [0][11]
[3212ms]
<br>* 쇼다운 상태 진입: [턴:0]
<br>* 커뮤니티 카드 딜: H(♣A♠J) C (♣9♠A♣10) (<font color='red'>♥</font>8) (<font color='red'>♥</font>J)
# 공베팅 반환 [334,855원]<br>* 종료: WinMoney[0원] Credit[335,522원]
<br>* 결과: 패배 [족보:A J 투 페어] [카드:♠A ♣A <font color='red'>♥</font>J ♠J ♣10]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-120,145</font></p></td>
											<td style="word-break:break-all"><p align="right">335,522</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>iuwjrjjff</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>A <font color='red'>◆</font>5 [A 탑]  (다이) 
<br>* 시작 : [StageNo:86576952] [Credit:87,391원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[iuwjrjjff]
<br>* 앤티: -1,000원(86,391원)
<br>* 홀 카드딜: <font color='red'>◆</font>A(13) <font color='red'>◆</font>5(17) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(<font color='red'>◆</font>A <font color='red'>◆</font>5)]
<br>* 베팅: 콜 -1,000원(85,391원) - 베팅순서: [0][3]
[1216ms]
<br>* 베팅: 다이 [0](85,391원) - 베팅순서: [0][10]
[174ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>A<font color='red'>◆</font>5) C (♣9♠A♣10) (<font color='red'>♥</font>8) (<font color='red'>♥</font>J)
<br>* 종료: WinMoney[0원] Credit[85,391원]
<br>* 결과: 패배 [족보:A 탑] [카드:<font color='red'>◆</font>A <font color='red'>◆</font>5] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">85,391</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>e3r5fg1fgfd</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣Q <font color='red'>◆</font>6 [Q 탑]  (다이) (Small Blind)
<br>* 시작 : [StageNo:86576952] [Credit:511,826원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[dkdffhakgfkgn]
<br>* 앤티: -1,000원(510,826원)
<br>* 홀 카드딜: ♣Q(50) <font color='red'>◆</font>6(18) [Q 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 탑(♣Q <font color='red'>◆</font>6)]
<br>* 베팅: [블라인드:SMALL] [금액:1,000원] [Creadit:509,826원]
<br>* 베팅: 다이 [0](509,826원) - 베팅순서: [0][8]
[1645ms]
<br>* 커뮤니티 카드 딜: H(♣Q<font color='red'>◆</font>6) C (♣9♠A♣10) (<font color='red'>♥</font>8) (<font color='red'>♥</font>J)
<br>* 종료: WinMoney[0원] Credit[509,826원]
<br>* 결과: 패배 [족보:Q 탑] [카드:♣Q <font color='red'>◆</font>6] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">509,826</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>Sweeety4</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠10 ♠2 [10 탑]  (다이) 
<br>* 시작 : [StageNo:86576952] [Credit:168,263원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[sweeety4]
<br>* 앤티: -1,000원(167,263원)
<br>* 홀 카드딜: ♠10(9) ♠2(1) [10 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:10 탑(♠10 ♠2)]
<br>* 베팅: 다이 [0](167,263원) - 베팅순서: [0][1]
[4329ms]
<br>* 커뮤니티 카드 딜: H(♠10♠2) C (♣9♠A♣10) (<font color='red'>♥</font>8) (<font color='red'>♥</font>J)
<br>* 종료: WinMoney[0원] Credit[167,263원]
<br>* 결과: 패배 [족보:10 탑] [카드:♠10 ♠2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">167,263</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>gkjasdb</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>Q ♣5 [Q 탑]  (다이) 
<br>* 시작 : [StageNo:86576952] [Credit:200,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:200]
<br>* NICKNAME:[gkjasdb]
<br>* 앤티: -1,000원(199,000원)
<br>* 홀 카드딜: <font color='red'>♥</font>Q(37) ♣5(43) [Q 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 탑(<font color='red'>♥</font>Q ♣5)]
<br>* 베팅: 다이 [0](199,000원) - 베팅순서: [0][7]
[326ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>Q♣5) C (♣9♠A♣10) (<font color='red'>♥</font>8) (<font color='red'>♥</font>J)
<br>* 종료: WinMoney[0원] Credit[199,000원]
<br>* 결과: 패배 [족보:Q 탑] [카드:<font color='red'>♥</font>Q ♣5] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">199,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>3t2gh2j4</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>9 <font color='red'>◆</font>9 ♣9 ♠A ♣10 <font color='red'>♥</font>8 <font color='red'>♥</font>J [9 트리플]  
<br>* 시작 : [StageNo:86576952] [Credit:120,145원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[3t2gh2j4]
<br>* 앤티: -1,000원(119,145원)
<br>* 홀 카드딜: <font color='red'>♥</font>9(34) <font color='red'>◆</font>9(21) [9 원 페어]
<br>* 턴 시작: [프리플랍(0)] [족보:9 원 페어(<font color='red'>♥</font>9 <font color='red'>◆</font>9)]
<br>* 베팅: [레이즈] (37,000원) Credit(82,145원) - 베팅순서: [0][6]
[3912ms]
<br>* 베팅: [올인] (82,145원) Credit(0원) - 베팅순서: [0][12]
[4375ms]
<br>* 쇼다운 상태 진입: [턴:0]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>9<font color='red'>◆</font>9) C (♣9♠A♣10) (<font color='red'>♥</font>8) (<font color='red'>♥</font>J)
<br>* 종료: WinMoney[237,025원] Credit[237,025원]
<br>* 결과: 승리 [족보:9 트리플] [카드:♣9 <font color='red'>♥</font>9 <font color='red'>◆</font>9 ♠A <font color='red'>♥</font>J]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>116,880</font></p></td>
											<td style="word-break:break-all"><p align="right">237,025</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>ngjyl9xfq2</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>Q ♣J [Q 탑]  (다이) (Big Blind)
<br>* 시작 : [StageNo:86576952] [Credit:200,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:200]
<br>* NICKNAME:[v32rfgdsg]
<br>* 앤티: -1,000원(199,000원)
<br>* 홀 카드딜: <font color='red'>◆</font>Q(24) ♣J(49) [Q 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 탑(<font color='red'>◆</font>Q ♣J)]
<br>* 베팅: [블라인드:BIG] [금액:1,000원] [Creadit:198,000원]
<br>* 베팅: 다이 [0](198,000원) - 베팅순서: [0][9]
[1219ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>Q♣J) C (♣9♠A♣10) (<font color='red'>♥</font>8) (<font color='red'>♥</font>J)
<br>* 종료: WinMoney[0원] Credit[198,000원]
<br>* 결과: 패배 [족보:Q 탑] [카드:<font color='red'>◆</font>Q ♣J] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">198,000</p></td>
											
										</tr>

									 </table>
								</td>
                            </tr>






						
                        </table>
                        
                        <table cellpadding="0" cellspacing="0" border="0" width="100%">
                            <tr>
                                <td width="100%" align="center" style="padding:1rem 0; background:#fdfdfd;">
                                	<input type="button" value="닫기" onclick="self.close()" class="btn-history-back">
                                </td>
                            </tr>
                        </table>
                        
		</div>
                        
	</div>




<form name="form1" method="Post">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="logno" value="15-2-86576952">
<input type="hidden" name="regdate_f" value="">
<input type="hidden" name="user_id">
<input type="hidden" name="userid">
<input type="hidden" name="rebateid">
<input type="hidden" name="topid">
<input type="hidden" name="infotype">
<input type="hidden" name="money">
</form>

<form name="form3" method="post">
<input type="hidden" name="idx">
<input type="hidden" name="p_logno">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="page" value="">

<input type="hidden" name="s_regdate_y1" value="">
<input type="hidden" name="s_regdate_m1" value="">
<input type="hidden" name="s_regdate_d1" value="">
<input type="hidden" name="s_regdate_h1" value="">
<input type="hidden" name="game_id" value="">
<input type="hidden" name="s_regdate" value="">
</form>


