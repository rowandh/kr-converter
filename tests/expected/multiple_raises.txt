PokerStars Hand #15286576952:  Hold'em No Limit ($1000/$1000) - 2024/12/27 19:20:56 KST
Table 'Table 1' 9-max Seat #7 is the button
Seat 1: Sweeety4 ($168263 in chips)
Seat 2: ftxijbnridn ($311675 in chips)
Seat 3: iuwjrjjff ($87391 in chips)
Seat 4: woeirvkd ($200000 in chips)
Seat 5: sdhewrj35 ($455667 in chips)
Seat 6: 3t2gh2j4 ($120145 in chips)
Seat 7: gkjasdb ($200000 in chips)
Seat 8: e3r5fg1fgfd ($511826 in chips)
Seat 9: ngjyl9xfq2 ($200000 in chips)
Sweeety4: posts the ante $1000
ftxijbnridn: posts the ante $1000
iuwjrjjff: posts the ante $1000
woeirvkd: posts the ante $1000
sdhewrj35: posts the ante $1000
3t2gh2j4: posts the ante $1000
gkjasdb: posts the ante $1000
e3r5fg1fgfd: posts the ante $1000
ngjyl9xfq2: posts the ante $1000
e3r5fg1fgfd: posts small blind $1000
ngjyl9xfq2: posts big blind $1000
*** HOLE CARDS ***
Sweeety4: folds
ftxijbnridn: folds
iuwjrjjff: calls $1000
woeirvkd: folds
sdhewrj35: raises $9000 to $10000
3t2gh2j4: raises $27000 to $37000
gkjasdb: folds
e3r5fg1fgfd: folds
ngjyl9xfq2: folds
iuwjrjjff: folds
sdhewrj35: raises $417000 to $454000
3t2gh2j4: calls $82145 and is all-in
Uncalled bet ($334855) returned to sdhewrj35
*** FLOP *** [9c As Tc]
*** TURN *** [9c As Tc] [8h]
*** RIVER *** [9c As Tc] [8h] [Jh]
3t2gh2j4 collected $237025 from pot
*** SUMMARY ***
Total pot $250290 | Rake $13265
Board [9c As Tc 8h Jh]
Seat 1: Sweeety4 mucked [Ts 2s]
Seat 2: ftxijbnridn mucked [8d 5s]
Seat 3: iuwjrjjff mucked [Ad 5d]
Seat 4: woeirvkd mucked [6c 2c]
Seat 5: sdhewrj35 showed [Ac Js] and lost
Seat 6: 3t2gh2j4 showed [9h 9d] and won ($237025)
Seat 7: gkjasdb mucked [Qh 5c]
Seat 8: e3r5fg1fgfd mucked [Qc 6d]
Seat 9: ngjyl9xfq2 mucked [Qd Jc]